ONSdigital/eq-author-app
# EAR-1574 List back end

Update to backend to support list component.

To Test:
Spin up the front and backend,
Use Apollo dev tools in the browser to send graphql calls.
Check -

- Create list
- Delete list
- update list
- Add answer to list
- Update answer on list
- Delete answer from list.
